### ISSUE TITLE
Digits in target temperature for thermostat 

### ISSUE THREAD

**Issue by mbierman** [oznu/homebridge-config-ui-x#741]
**Describe The Bug:**
The digits for Target temperature are too many decimal places. “60.980000000000004°”

**To Reproduce:**
1. Go to accessories. 
1. Look at a thermostat.

**Expected behavior:**
Show 1 decimal place for the target temperature.

**Logs:**

```
Show the Homebridge / Homebridge Config UI X logs here.
```

**Homebridge Config:**

```
{
    "bridge": {
        "name": "Homebridge Synology",
        "username": "",
        "port": 53036,
        "pin": ""
    },
    "accessories": [
        {
            "accessory": "PowerlossIFTTTNotifier",
            "name": "PowerlossIFTTTNotifier",
            "IFTTTkey": "",
            "IFTTTservice": "",
            "IFTTTvalue2": "Homebridge started at",
            "IFTTTvalue3": false
        },
        {
            "accessory": "Synology",
            "name": "Pigpen",
            "ip": "",
            "mac": "",
            "port": "5001",
            "secure": true,
            "account": "",
            "password": "",
            "version": 6,
            "timeout": 15000,
            "disabled": [
                ""
            ],
            "doPolling": true,
            "pollingInterval": 120
        },
        {
            "accessory": "neurio",
            "name": "neurio",
            "location": "",
            "username": "",
            "password": "",
            "options": {
                "ttl": 30,
                "verboseP": false
            }
        },
        {
            "name": "Garage Door",
            "username": "",
            "password": "",
            "accessory": "Chamberlain"
        },
        {
            "accessory": "HTTP-SWITCH",
            "name": "Flo",
            "switchType": "stateful",
            "pullInterval": 5000,
            "debug": false,
            "statusPattern": "true",
            "onUrl": {
                "url": "",
                "method": "GET"
            },
            "offUrl": {
                "url": "",
                "method": "GET"
            },
            "statusUrl": {
                "url": "",
                "method": "GET",
                "headers": {
                    "Accept": "application/json",
                    "Authorization": "Basic"
                }
            }
        },
        {
            "accessory": "AutomationCalendar",
            "name": "AutomationCalendar",
            "latitude": ,
            "longitude": 
        }
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "blue",
            "tempUnits": "f",
            "lang": "auto",
            "sessionTimeout": 28800,
            "log": {
                "method": "null"
            },
            "accessoryControl": {
                "debug": true
            },
            "platform": "config"
        },
        {
            "platform": "Nest",
            "name": "Nest",
            "access_token": ""
        },
        {
            "platform": "TuyaWebPlatform",
            "name": "Smart Life",
            "options": {
                "username": "",
                "password": "",
                "countryCode": "1",
                "platform": "smart_life",
                "pollingInterval": 120
            }
        },
        {
            "platform": "WeatherPlus",
            "units": "us",
            "interval": 5,
            "stations": [
                {
                    "nameNow": "Now",
                    "service": "openweathermap",
                    "key": "",
                    "locationGeo": [
                        37.35,
                        -121.96
                    ],
                    "locationCity": "",
                    "language": "en",
                    "compatibility": "both",
                    "forecast": [
                        0
                    ],
                    "conditionCategory": "detailed",
                    "now": true,
                    "extraHumidity": false,
                    "hidden": [
                        "Air Pressure",
                        "Observation Station",
                        "Observation Time",
                        "Snow"
                    ],
                    "tresholdCloudCover": 30,
                    "tresholdUvIndex": 4,
                    "tresholdWindSpeed": 10
                }
            ]
        }
    ]
}
```

**Screenshots:**
![B410D1F8-1688-4BA1-A752-DD1DBFB46137](https://user-images.githubusercontent.com/1205471/84306780-cae43000-ab10-11ea-9f0d-a197cd29c73c.png)

**Environment:**

* **Node.js Version**: v12.180
* **NPM Version**: v6.14.5
* **Homebridge Version**: V1.1.0
* **Homebridge Config UI X Version**: v4.22.0
* **Operating System**: Docker
* **Process Supervisor**: Docker

**Comment by oznu** [comment]
I'll fix this.

If you swap to normal units (Celsius) this wouldn't be an issue 😆

**Comment by mbierman** [comment]
LoL Thanks Oznu!

**Comment by oznu** [comment]
Fixed in 4.23.0
